pybaseball/lahman.py

```diff
@@ -1,9 +1,11 @@
-import requests
-from zipfile import ZipFile
 import os
-import pandas as pd
+import zipfile
 from io import BytesIO
-from typing import Optional
+from typing import IO, Optional, Tuple, Union
+from zipfile import ZipFile
+
+import pandas as pd
+import requests
```

Comment:
This should be untouched

Reply:
I'll undo. Sorry. Must have organized these imports.